client/state/happychat/middleware.js

```diff
@@ -135,6 +137,12 @@ const sendBrowserInfo = ( connection, siteUrl ) => {
 	connection.info( msg );
 };
 
+export const checkForRecentActivity = ( connection, store ) => {
```

Comment:
The name of this function kind of suggests that it won't have any side effects. How about something like `connectIfRecentlyActive`?